[{"ContainerToolsException.Message":"Docker command failed with exit code 1. #1 [internal] load build definition from dockerfile #1 sha256:fae2ba4dfff34c6496961c063a79e50c7dee613cf99b104cd99ca288ad494a06 #1 transferring dockerfile: 2B done #1 DONE 0.0s failed to solve with frontend dockerfile.v0: failed to read dockerfile: open /var/lib/docker/tmp/buildkit-mount344176138/dockerfile: no such file or directory","ContainerToolsException.ErrorCode":"CTC1014","ContainerToolsException.StackTrace":"   at Microsoft.VisualStudio.Containers.Tools.Common.Client.DockerClient.<BuildImageAsync>d__4.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.VisualStudio.Containers.Tools.Shared.ContainerLaunchManager.<RebuildImageAsync>d__12.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)\r\n   at Microsoft.VisualStudio.Containers.Tools.Shared.ContainerLaunchManager.<BuildAndLaunchForDockerfileAsync>d__7.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.VisualStudio.Containers.Tools.Shared.ContainerLaunchManager.<BuildAndLaunchAsync>d__5.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)\r\n   at Microsoft.VisualStudio.Containers.Tools.Tasks.ContainerBuildAndLaunch.<ExecuteCoreAsync>d__176.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.VisualStudio.Containers.Tools.Tasks.ContainerBuildTask.<ExecuteAsync>d__26.MoveNext()","ContainerToolsException.HResult":"-2146233088","ContainerToolsException.FaultingUserFile.FilePath":"dockerfile","ContainerToolsException.FaultingUserFile.LineStart":"","ContainerToolsException.FaultingUserFile.ColumnStart":"","ContainerToolsException.FaultingUserFile.LineEnd":"","ContainerToolsException.FaultingUserFile.ColumnEnd":"","ContainerToolsException.CommandLine.Program":"docker","ContainerToolsException.CommandLine.TelemetrySafeArguments":"build","ContainerToolsException.CommandLine.ExitCode":"1","ContainerToolsException.CommandLine.ErrorOutput":"#1 [internal] load build definition from dockerfile\r\n#1 sha256:fae2ba4dfff34c6496961c063a79e50c7dee613cf99b104cd99ca288ad494a06\r\n#1 transferring dockerfile: 2B done\r\n#1 DONE 0.0s\r\nfailed to solve with frontend dockerfile.v0: failed to read dockerfile: open /var/lib/docker/tmp/buildkit-mount344176138/dockerfile: no such file or directory"}]